mumd/src/network/udp.rs

```diff
@@ -9,7 +9,7 @@ use mumble_protocol::crypt::ClientCryptState;
 use mumble_protocol::ping::{PingPacket, PongPacket};
 use mumble_protocol::voice::VoicePacket;
 use mumble_protocol::Serverbound;
-use std::collections::HashMap;
```

Comment:
The import of `sync::RwLock` should be grouped below with the import of `std::sync::Arc` and not with the import of `std::collections::HashMap`.